src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_controls.erl

```diff
@@ -48,6 +48,22 @@ init(Config) ->
 init_resource_state(_Config) ->
     {ok, #control_state{}}.
 
+service_available(Req, State) ->
+    case oc_chef_wm_base:service_available(Req, State) of
+        {true, Req1, State1} ->
+            ActionsEnabled = envy:get(oc_chef_wm, enable_actions, false, boolean),
+                case ActionsEnabled of
```

Comment:
Slightly odd spacing here, `case` should start at the same column as `ActionsEnabled` in the line above.